src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/session/GetAvailableSessionsTest.kt

```diff
@@ -2240,7 +2240,7 @@ class GetAvailableSessionsTest : IntegrationTestBase() {
     // Given
     val prisonerId = "A1234AA"
     val associationPrisonerId = "B1234BB"
-    val validFromDate = this.getNextAllowedDay().minusMonths(6)
+    val validFromDate = this.getNextAllowedDay().minusWeeks(6)
```

Comment:
changed this to weeks to ensure the day of week does not change as session template below has the dayOfWeek as  validFromDate.dayOfWeek